apps/server/src/modules/learnroom/controller/dto/course.query.params.ts

```diff
@@ -12,4 +12,11 @@ export class CourseQueryParams {
 		enum: CommonCartridgeVersion,
 	})
 	version!: CommonCartridgeVersion;
+
+	@IsString()
+	@IsOptional()
+	@ApiPropertyOptional({
+		description: 'The ids of topics which should be exported separated by a comma.',
+	})
+	topics?: string;
```

Comment:
Why not a list of strings? If it's a list of strings you don't need to split and process the result.

```suggestion
	topics: string[];
```